go.mod

```diff
@@ -0,0 +1,15 @@
+module github.com/neevaco/antch
```

Comment:
Optional: Do you want to split out the `go.mod` changes into a separate PR, or at least attempt to upstream that separately?

Reply:
done .. removed from this PR